livre/eq/Window.cpp

```diff
@@ -69,10 +69,12 @@ struct Window::Impl
                         pipe->getFrameData()->getVRParameters().getMaxGPUCacheMemoryMB();
         _textureCache.reset( new TextureCache( node->getTextureDataCache(),
                                                maxGpuMemory * LB_1MB, GL_LUMINANCE8 ));
-
-        const size_t computeThreads = 4;
+        const size_t renderThreads = 2;
+        const size_t computeThreads = 2;
         const size_t uploadThreads = 4;
```

Comment:
For benchmarking purposes you could set this variables from a config file or environmental variables without requiring them in the interface for RenderPipeline. If you want to go smart and decide the best configuration at runtime I still think that's something that belongs to the RenderPipeline implementation.